### ISSUE TITLE
Internet not working!

### ISSUE THREAD

**Issue by saurabhcs0097** [securestate/king-phisher#197]
Hi 

I have installed KingPhiser in my Kali machine, but after installing internet is not working. I tried many solutions but didnt work for me.
Same has happened on another Kali machine when I installed King Phisher on it.
can you please help !

**Comment by wolfthefallen** [comment]
Hi,

King Phisher makes no changes to the operating system that would effect its internet or network connection. We have been unable to reproduce this.

**Comment by zeroSteiner** [comment]
I'm going to close this issue as it is pretty obviously not related to the King Phisher project. I recommend you look at your OS settings and perform basic network troubleshooting. If once this is complete you have confirmed that the issue was in fact due to King Phisher, please feel free to open a new issue.
